Week1/homework/style.css

```diff
@@ -1 +1,56 @@
-/* add your styling here */
\ No newline at end of file
+/* add your styling here */
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  background-image: url(./img/book.jpg);
+  text-align: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: white;
+}
+.bookTitlesList {
+  list-style-type: none;
+  width: 80%;
+  margin: 20px auto;
+  text-align: center;
+  background-color: #333 0.5;
+  font-family: cursive;
+}
+h1 {
+  font-style: italic;
+  text-align: center;
+  margin: 20px auto;
+  background-color: #6aabf5;
+  padding: 10px;
+}
+.bookInformationList {
+  display: flex;
+  flex-flow: row wrap;
+  background-color: #5a4446;
+  opacity: 0.9;
+  margin: 20px auto;
+  align-items: center;
+  padding-bottom: 40px;
+}
+.book-information {
+  width: calc(50% - 80px);
+  margin: 40px;
+  opacity: 1;
+  overflow: hidden;
+  box-shadow: black 3px 3px 4px;
+}
+.book-information h1 {
```

Comment:
Your `h1` elements already have a class assigned to them (`book-title`) - I'd suggest using that class name here instead.

Reply:
Thank you for your time and nice comment. css line-45 solved;